tox.ini

```diff
@@ -13,36 +13,32 @@ extras =
     test
 
 commands =
-    {envbindir}/buildout -c {toxinidir}/py3.cfg buildout:directory={envdir} bootstrap
-    {envbindir}/buildout -c {toxinidir}/py3.cfg buildout:directory={envdir} annotate
-    {envbindir}/buildout -c {toxinidir}/py3.cfg buildout:directory={envdir} install test
-    {envbindir}/buildout -c {toxinidir}/py3.cfg buildout:directory={envdir}
+    {envbindir}/buildout -c {toxinidir}/py3.cfg buildout:directory={envdir} buildout:sources-dir={toxinidir}/src bootstrap
+    {envbindir}/buildout -c {toxinidir}/py3.cfg buildout:directory={envdir} buildout:sources-dir={toxinidir}/src annotate
+    {envbindir}/buildout -c {toxinidir}/py3.cfg buildout:directory={envdir} buildout:sources-dir={toxinidir}/src install test
+    {envbindir}/buildout -c {toxinidir}/py3.cfg buildout:directory={envdir} buildout:sources-dir={toxinidir}/src
     coverage run {envbindir}/test -v1 --auto-color {posargs}
 
 setenv =
     COVERAGE_FILE=.coverage.{envname}
 
-deps =
-    setuptools==39.0.1
-    zc.buildout==2.11.2
+deps = -r{toxinidir}/requirements.txt
```

Comment:
yes in the current situation the better way on pinning the requirements, in the long run it should be:

    deps = setuptools zc.buildout -c https://dist.plone.org/bootstrap-buildout-constraints.txt